src/Pmi/WorkQueue/WorkQueue.php

```diff
@@ -520,4 +522,15 @@ public function getActivityStatus($participant)
                 return '';
         }
     }
+
+    public function displayProgramUpdate($participant)
+    {
+        if ($participant->consentCohort === 'COHORT_1' || $participant->consentCohort === 'COHORT_3') {
```

Comment:
Should this be !== 'COHORT_2'? This only applies to Cohort 2, right? By changing this around, any future cohorts wouldn't be included, which seems like what we want.